modin/pandas/test/utils.py

```diff
@@ -54,6 +54,10 @@
 # Directory for storing I/O operations test data
 IO_OPS_DATA_DIR = os.path.join(os.path.dirname(__file__), "io_tests_data")
 
+# Once a partition has more than this many elements, Modin uses another
+# partition.
+min_num_elements_to_start_new_partition = 32
```

Comment:
What I had in mind is we should have a variable to specify default/minimal partition size, put it in `modin/core/storage_formats/pandas/utils.py` and use it where possible. One of the places would be [here](https://github.com/modin-project/modin/blob/72a3e51aeeae3465a1a3076d962aef95af689dde/modin/core/storage_formats/pandas/utils.py#L41) as well, for example.

Reply:
I wonder if we should have this (the parameter in the splitting function) as a configurable variable in `modin.config` so users could tune Modin behaviour specifically to their workloads...

Reply:
Yes, it would probably be a better way.